server/publications/collections/cart.js

```diff
@@ -67,6 +67,33 @@ Meteor.publish("Cart", function (sessionId, userId) {
   return Cart.find(cartId);
 });
 
+
```

Comment:
@machikoyasuda It'd be useful to have all the publications documented using jsdoc.  I think I just never started them because I wasn't sure the best format. Maybe you could/doc/suggest the best format?